src/main/java/racingcar/domain/RacingGame.java

```diff
@@ -0,0 +1,27 @@
+package racingcar.domain;
+
+import java.util.List;
+
+import common.NumberGenerator;
+
+public class RacingGame implements Game {
+    List<Vehicle> vehicles;
+    NumberGenerator numberGenerator;
```

Comment:
```suggestion
    private List<Vehicle> vehicles;
    private NumberGenerator numberGenerator;
```

필드 접근제한자에 익숙치 않으신 것 같아요.
객체의 정보 은닉과 캡슐화를 위해 기계적으로 private을 사용하시고 필요 시에 조금씩 개방하시는게 어떤가요?